Assets/ZSSRichTextEditor.js

```diff
@@ -1860,6 +2182,39 @@ ZSSField.prototype.sendImageTappedCallback = function( imageNode ) {
     setTimeout(function() { thisObj.callback('callback-image-tap', joinedArguments);}, 500);
 }
 
+ZSSField.prototype.sendVideoTappedCallback = function( videoNode ) {
+    var videoId = "";
+    if ( videoNode.hasAttribute( 'data-wpid' ) ){
+        videoId = videoNode.getAttribute( 'data-wpid' )
+    }
+    var arguments = ['id=' + encodeURIComponent( videoId ),
+                     'url=' + encodeURIComponent( videoNode.src )];
+    
+    var joinedArguments = arguments.join( defaultCallbackSeparator );
+    
+    var thisObj = this;
+    
+    // WORKAROUND: force the event to become sort of "after-tap" through setTimeout()
+    //
+    setTimeout(function() { thisObj.callback('callback-video-tap', joinedArguments);}, 500);
```

Comment:
@SergioEstevao what is this code working around? If it was not here, does a touch event exhibit strange behavior?